packages/extension/src/config.ts

```diff
@@ -77,6 +77,11 @@ export const EmbedChainInfos: ChainInfo[] = [
     ],
     coinType: 118,
     features: [],
+    gasPriceStep: {
+      low: 4000000000,
+      average: 4000000000,
+      high: 5000000000,
```

Comment:
Since we want to set the minimal value to 5 nano FET. I think that we set the low and average values to this minimum  value and then have the high fee as double that. If we don't do that then the low and average fee values would be automatically rejected :grin: 

```suggestion
      low: 5000000000,
      average: 5000000000,
      high: 10000000000,
```
Can you also update this on all the network configurations - this will hopefully allow us to get a head start on the fee changes

Reply:
It makes total sense to stick with the minimum as the minimum, this just means that we need to change the gas amount in the stake delegation transaction in ledger-explorer from 160000 to 200000 atestfet If the goal is to have the typical delegation transaction fee be 0.0008 TESTFET.

If we want the fee upper limit for that transaction to be 0.001 TESTFET, then it seems like we should set the high fee type to 6250000000. 10000000000 would mean users may submit transactions with a fee of 0.0016 TESTFET, well above the "upper limit".

Reply:
Sorry for the delay meeting central - see my other comment on the ledger-explorer repo but we basically need to keep the gas at 200000 otherwise we risk some txs not going through.

Can we also apply this gas step changes to all the registered networks as well i.e. `fetchhub-2`, `andromeda-1` & `stargateworld-3`